portal/server/controllers/dashboard/deals/index.test.js

```diff
@@ -135,6 +136,40 @@ describe('controllers/dashboard/deals', () => {
     });
   });
 
+  describe('getAllDealsData with blank session.sortBy', () => {
+    it('should calls api.allDeals with default sort query if session.sortBy is blank', async () => {
+      mockReq.session.sortBy = null;
+
+      await getAllDealsData(
+        'mock-token',
+        mockReq.session.user,
+        mockReq.session.dashboardFilters,
+        mockReq.params.page,
+        mockReq.session.sortBy,
```

Comment:
Optional chaining perhaps, as by this point `sortBy` will not exists.

Reply:
but that is the point of this test? I am testing sending a blank/null sortBy to the function to check that it is replaced by the default sort

Reply:
discussed